web/themes/new_weather_theme/assets/sass/components/quick-forecast.scss

```diff
@@ -0,0 +1,299 @@
+
+    justify-content: center;
+}
+
+.wx-quick-forecast-item-bottom {
+    @include u-font("body", "sm");
+    @include grid-col(12);
+
+    color: color("primary-darker");
+    text-align: left;
+    margin-top: units(1);
+}
+
+.wx-quick-forecast-item {
+    @include grid-col(12);
+    @include grid-row;
+
+    padding: units(1.5);
+    text-decoration: none;
+    border-bottom: 1px solid color("base-lighter");
+}
+
+.wx-quick-high,
+.wx-quick-low {
+    @include u-font("mono", "sm");
+    @include grid-col(6);
+
+    margin-bottom: units(0.5);
+}
+
+.wx-quick-label {
+    display: none;
+    text-transform: uppercase;
+    color: color("base-dark");
+}
+
+.wx-quick-temp {
+    display: flex;
+    flex-flow: row nowrap;
+    align-items: flex-start;
+    color: color("primary-darker");
+
+    @include u-font("body", "sm");
+
+    span {
+        @include u-font("body", "3xs");
+    }
+}
+
+.wx-alert-icon {
+    width: units(3);
+    height: units(3);
+
+    &[data-alert-level] {
+        fill: color("base");
+    }
+
+    &[data-alert-level="watch"] {
+        fill: color("yellow-50v");
+    }
+
+    &[data-alert-level="warning"] {
+        fill: color("red-50");
+    }
+}
+
+
+.wx-quick-forecast-header {
+    @include u-font("mono", "sm");
+
+    text-transform: uppercase;
+    width: 100;
+    color: color("primary-darker");
+    padding: units(1.5);
+
+    @include grid-row;
+}
+
+.wx-quick-forecast-header-top {
+    @include grid-row;
+    @include grid-col;
+}
+
+.wx-quick-forecast-header-bottom {
+    @include grid-row;
+
+    display: none;
+}
+
+.wx-quick-forecast-header-date {
+    @include grid-col;
+}
+
+.wx-quick-forecast-header-condition {
+    @include grid-col;
+
+    display: flex;
+    justify-content: space-around;
+}
+
+.wx-quick-forecast-header-highlow {
+    @include grid-col;
+    @include grid-row;
+
+    span {
+        @include grid-col(6);
+    }
+}
+
+
+/* Desktop styles
+-------------------------------------------- */
+@include at-media("desktop") {
+    .wx-quick-forecast-header {
+        display: none;
+    }
+    
+    .wx-quick-forecast {
+        @include grid-row;
+        @include grid-col(12);
+
+        flex-wrap: nowrap;
+
+        .wx-alert-icon {
+            width: units(4);
+            height: units(4);
+
+            &:first-child {
+                margin-right: units(0.5);
+            }
+        }
+    }
+
+    .wx-quick-forecast-item {
+        @include grid-col;
+        @include grid-row;
+
+        border-right: 1px solid color("base-lighter");
+
+        &:last-child {
+            border-right: none;
+        }
+
+        hr {
+            display: block;
+            border: none;
+            border-bottom: 1px solid color("base-lighter");
+            width: 100%;
+            margin: 0;
+        }
+
+        time {
+            display: block;
+            text-align: center;
+
+            @include grid-col(12);
+
+            span:last-child {
+                text-decoration: none;
```

Comment:
Looking at it again, probably should keep the underline hint for desktop too, can take this one out